app/controllers/concerns/sufia/files_controller_behavior.rb

```diff
@@ -64,6 +64,7 @@ def load_resource_from_solr
 
     # routed to /files/new
     def new
+      set_variables_for_new_form
```

Comment:
I never mind adding more tests.  I threw in a couple of others just to be more complete.